lib/commands/debug/helper.js

```diff
@@ -0,0 +1,143 @@
+const { execSync } = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const nodemon = require('nodemon');
+const CliError = require('@src/exceptions/cli-error');
+
+module.exports = {
+    getSkillInvocationInfo,
+    initiateDebugging
+};
+
+function getPythonDebugAdapterPath() {
+    let localDebuggerPath;
+    const sitePkgUint8Array = execSync('python3 -c "import site; print(site.getsitepackages())"');
+    let sitePkgLocationsStr = new TextDecoder('utf8').decode(sitePkgUint8Array);
+    // Preprocessing the string to get site locations for searching
+    // https://docs.python.org/3/library/site.html#site.getsitepackages gives an array of strings
+    // eg: "['sitePkg-A', 'sitePkg-B']", and we need to preprocess to get each location
+    sitePkgLocationsStr = sitePkgLocationsStr.replace('[', '').replace(']', '').trim();
+    const sitePkgLocations = sitePkgLocationsStr.split(',');
+    for (let sitePkgLocation of sitePkgLocations) {
+        // Remove extra quotes and white spaces
+        sitePkgLocation = sitePkgLocation.replace(/['"]+/g, '').trim();
+        localDebuggerPath = path.join(sitePkgLocation, 'ask_sdk_local_debug', 'local_debugger_invoker.py');
+        if (fs.existsSync(localDebuggerPath)) {
+            break;
```

Comment:
when python debugger starts there is no any messages in the console similar to node debugger. It is probably specific to python debugger module.